src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,118 @@
 Release History
 ===============
 
+2.36.0
+++++++
+
+**ACR**
+
+* `acr task run`: Add `--no-format` option (#21983)
+* `acr task logs`: Add `--no-format` option (#21983)
+* `acr taskrun logs`: Add `--no-format` option (#21983)
+
+**AKS**
+
+* `az aks create`: Add `--nat-gateway-managed-outbound-ip-count` and `--nat-gateway-idle-timeout` to support nat gateway integration (#21623)
+* `az aks create`: Add `managedNATGateway` and `userAssignedNATGateway` to supported outbound type (#21623)
+* `az aks check-acr`: Bump canipull to 0.0.4-alpha to skip location check if cname returns only privatelink (#22092)
+
+**AMS**
+
+* Update AMS Module to 9.0 Python sdk (#22056)
```

Comment:
```suggestion
* `az ams asset-track create`: Add command to create an asset track (#22056)
* `az ams asset-track show`: Add command to show an asset track (#22056)
* `az ams asset-track list`: Add command to list all tracks under an asset (#22056)
* `az ams asset-track update`: Add command to update the parameters of a track (#22056)
* `az ams asset-track update-data`: Add update-data command to refresh the server in case track file was updated (#22056)
* `az ams asset-track delete`: Add command to delete track (#22056)
* `az ams streaming-endpoint get-skus`: Add command to get skus under a streaming endpoint (#22056)
```